.eslintignore

```diff
@@ -2,3 +2,5 @@
 **/vendor/**/*.js
 django/contrib/gis/templates/**/*.js
 node_modules/**.js
+Gruntfile.js
```

Comment:
Is there a reason to exclude `Gruntfile.js`? I recognize we have done so in the past, but maybe it was an oversight. It is JavaScript code to be maintained just like other files, so, IMO, might as well include it during lint.

Reply:
I removed this and fixed the lint errors. 👍